[BUG] overnight coverage test fail: PeriodogramTransformer' object has no attribute '_tags_dynamic'

### Describe the bug

aeon/base/estimator/hybrid/tests/test_base_rist.py::test_rist_soft_dependencies fail of test without numba:



### Steps/Code to reproduce the bug

https://github.com/aeon-toolkit/aeon/actions/runs/11246848175/job/31269353618

### Expected results

should pass the test

### Actual results

FAILED aeon/base/estimator/hybrid/tests/test_base_rist.py::test_rist_soft_dependencies - AttributeError: 'PeriodogramTransformer' object has no attribute '_tags_dynamic'

### Versions

_No response_

--- a/aeon/transformations/collection/_periodogram.py
+++ b/aeon/transformations/collection/_periodogram.py
@@ -71,10 +71,10 @@ class PeriodogramTransformer(BaseCollectionTransformer):
         self.use_pyfftw = use_pyfftw
         self.n_jobs = n_jobs
 
+        super().__init__()
         if use_pyfftw:
             self.set_tags(**{"python_dependencies": "pyfftw"})
 
-        super().__init__()
 
     def _transform(self, X, y=None):
         threads_to_use = check_n_jobs(self.n_jobs)

--- a/aeon/transformations/collection/_periodogram.py
+++ b/aeon/transformations/collection/_periodogram.py
@@ -75,7 +75,6 @@ class PeriodogramTransformer(BaseCollectionTransformer):
         if use_pyfftw:
             self.set_tags(**{"python_dependencies": "pyfftw"})
 
-
     def _transform(self, X, y=None):
         threads_to_use = check_n_jobs(self.n_jobs)
 

